### PR TITLE
Search view sim#42

### PR DESCRIPTION
**Pull request by sruble**

### CODE REVIEW

**Comment by sruble** [comment]
redefined DOD: when the viewer can see options to search the database

**Comment by sruble** [comment]
this can be merged to the Develop branch as soon as it's been reviewed

**Review by weberwilliamb — APPROVED** [review]
Not totally sure on the foreach loops.

**Comment by sruble** [comment]
> Not totally sure on the foreach loops.
the foreach loop is incorrect, I'm still working on it, that'll be pull request / issue #41.  Did you feel ok wth the search options in the SearchPhotoPoints.cshtml file?

File: photo_points/Views/Admin/SearchPhotoPoints.cshtml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+@addTagHelper *,Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    Layout = "_Layout";
+    ViewBag.Title = "Search";
+}
+
+<h3>Search Photos</h3>
+
+<table>
```
Comment (weberwilliamb) [inline photo_points/Views/Admin/SearchPhotoPoints.cshtml:12]: Search options are good. I might add whether the photo is approved or rejected, but that is not as important as what you already have in the weight of the search.

Comment (sruble) [reply photo_points/Views/Admin/SearchPhotoPoints.cshtml:12]: good point about the approval type . I'll add the option to the UI.

File: photo_points/Services/AdminReviewServices.cs [hunk 2]
```diff
@@ -31,17 +32,16 @@ public IQueryable<Capture> GetAllCaptures()
...
-        public IEnumerable<Capture> GetUnnapprovedCaptures()
-        {
-            return _fakeAdminRepo.GetCaptures();
-            ////Eric: is the line above correct? everything after the dot . was missing and I filled it in
-        }
-
         public IEnumerable<Capture> GetUnapprovedCaptures()
         {
-            throw new NotImplementedException();
+            return _fakeAdminRepo.GetAllUnapproved();
```
Comment (RoEric) [inline photo_points/Services/AdminReviewServices.cs]: I changed this line with the merge of my branch into develop.  There is no GetAllUnapproved() method in the repository anymore. Instead, GetCaptures() is called to return a collection of all the captures which is then filtered to only return the ones where the approval property type is pending.

**Review by RoEric — CHANGES_REQUESTED** [review]

File: photo_points/Services/AdminReviewServices.cs (hunk 2 above)
Comment (sruble) [reply photo_points/Services/AdminReviewServices.cs]: thanks for letting me know, I'll be sure to make a note of it when it comes time to merge my branch into develop.

**Review by Sara-webaplication — APPROVED** [review]

**Review by RoEric — APPROVED** [review]
I didn't see any issues with your work. The search options look fine to me.

**Pull request merged**